src/main/kotlin/com/squareup/cash/hermit/goland/GoSdkExtensions.kt

```diff
@@ -7,8 +7,11 @@ import com.intellij.openapi.project.Project
 import com.squareup.cash.hermit.HermitPackage
 
 fun HermitPackage.setSdk(project: Project): GoSdk {
-    val new = GoSdkImpl("file://" + this.path, this.version, this.path + "/src/runtime/internal/sys/zversion.go")
+    val new = GoSdkImpl(this.goURL(), this.version, this.path + "/src/runtime/internal/sys/zversion.go")
```

Comment:
should we use `Paths.get()` to handle slashes/platform inconsistencies?

Reply:
I might leave this refactoring for later, as this is old code.